### DISCUSSION TITLE
Can't get FanControl UI to display with this plugin

### DISCUSSION THREAD

**Discussion by Schnazzy** [EvanMulawski/FanControl.CorsairLink#35] [General]
I can't get FanControl to load with this plugin installed. I have the Corsair iCure H100i Elite Capellix which comes with the iCUE Commander Core fan controller. I'd love to get this working so I can get away from the iCUE software. Any help is appreciated!

Log.txt output follows:

4/10/2023 4:41:49 PM: [CorsairLink] DeviceManager: Found 2 Corsair device(s)
  name=CORSAIR iCUE Commander CORE, devicePath=\\?\hid#vid_1b1c&pid_0c1c&mi_01#7&2e89a725&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}
  name=CORSAIR iCUE Commander CORE, devicePath=\\?\hid#vid_1b1c&pid_0c1c&mi_00#7&61ef95c&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}

4/10/2023 4:41:49 PM: [CorsairLink] DeviceManager: Found 1 supported Corsair device(s)
  name=CORSAIR iCUE Commander CORE, devicePath=\\?\hid#vid_1b1c&pid_0c1c&mi_00#7&61ef95c&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}

**Comment by EvanMulawski** [comment]
Hi @Schnazzy, can you explain what is happening in more detail? Is Fan Control failing to start or show an error message? Is there anything else in the log file?

**Reply by Schnazzy** [reply]
I did a fresh install of FanControl (.net version) and confirmed it worked. I then shut down FanControl and extracted the .dll for the plugin to the plugins subfolder. I ran FanControl again and nothing appears to happen. The log file then shows the lines I pasted earlier...those are the only lines that are added to it each time I run FanControl with the plugin .dll file present. I've tried rebooted as well and re-running FanControl and I get the same result. If I rename the .dll file and run FanControl, it launches successfully with the UI displayed as expected

**Reply by EvanMulawski** [reply]
Is iCUE installed? If it is, exit/kill the iCUE program and ensure the "Corsair Service" service is stopped using the Services app.

**Reply by Schnazzy** [reply]
iCUE was installed at first, but I had uninstalled it and rebooted several times. Same results.

**Reply by EvanMulawski** [reply]
Can you try starting Fan Control with the plugin and then wait 5 minutes? Also check Task Manager to see if Fan Control remains running.

There definitely should be additional log entries if the device connection succeeded or failed. It's extremely odd that the logging just stops after listing the supported devices.

**Reply by Schnazzy** [reply]
I figured out a solution. Uninstalling the Citrix Workspace app as was suggested [here](https://forum.corsair.com/forums/topic/183783-hardware-suddenly-not-showing-up-in-icue/#comment-1045235) did the trick. Earlier today, I determined the issue wasn't specific to this FanControl plugin. I reinstalled iCUE and I was not able to see any of the Corsair hardware there either.
